SofaKernel/framework/sofa/helper/system/PluginManager.cpp

```diff
@@ -87,12 +87,18 @@ PluginManager::~PluginManager()
 void PluginManager::readFromIniFile(const std::string& path)
 {
     std::ifstream instream(path.c_str());
-    std::string pluginPath;
-
-    while(std::getline(instream,pluginPath))
+    std::string pluginPath, line, version;
+    while(std::getline(instream, line))
     {
+        if (line.empty()) continue;
+
+        std::istringstream is(line);
+        is >> pluginPath;
+        is >> version; // information not used for now
```

Comment:
Actually it already work with previous format (w/o version) but you're right about a deprecated message.
About the version thing in the filename, it is done already on Mac and Linux (e.g libFredPlugin.0.1.dylib is created and a symbolic link libFredPlugin points to it).
I guess you would like to load a specific version, e.g
CImgPlugin 0.5 then load CImgPlugin.0.5.dylib if not warning/error about the version, right ?
I think it should be the purpose of a future PR which will 
 - make version consistent between (all?) plugin's CMakefile.txt version (set_target_properties(${PROJECT_NAME} PROPERTIES VERSION ${PROJECT_VERSION})) and getVersion() in the initPluginXXXX.cpp
 - add version to generated Windows libraries (it does not, and not really possible with symbolic link a la Unix)
 - add checks and a version parameter to PluginManager loading functions